server/views/what-type-of-item-is-it.html

```diff
@@ -21,7 +21,7 @@
           }
       }) %}
 
-        <p class="govuk-body" id="introPara">Unless your item is to be sold or hired out to a qualifying museum, any replacement ivory must have been taken from an elephant before 1975 and used only to restore the item.</p>
+        <p class="govuk-body" id="introPara">Unless your item is to be sold or hired out to a qualifying museum, any replacement ivory must have been taken from an elephant before 1975 and added only for the purpose of restoring the item.</p>
```

Comment:
Add unit test to check for existence and value of "introPara" (after the unit test "should have the correct page heading")